platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -5,7 +5,6 @@ import { parse as parseSemVer, SemVer, Range } from "semver";
 
 export interface MainManifest {
```

Comment:
From what I have seen in (possibly only) the latest version of io-ts, the `lib` in the path might be redundant

> It does not look nice, anyway.